definitions/features/qpid_queues.rb

```diff
@@ -0,0 +1,44 @@
+class Features::QpidQueues < ForemanMaintain::Feature
+  metadata do
+    label :qpid_queues
+
+    confine do
+      qpid_tools_installed? && File.exist?('/etc/pki/katello/qpid_client_striped.crt')
+    end
+  end
+
+  def clear_all
+    queues_cleared = []
+    available_qpid_queues.each do |qname|
+      clear(qname)
+      queues_cleared << qname
+    end
+  ensure
+    queues_cleared
```

Comment:
Yes, your right.
Removed ensure block and added return value in rescue itself.
For return statement in ensure block, it was giving rubocop warning - 'Do not return from an ensure block'